src/hsd/dict.py

```diff
@@ -215,8 +215,16 @@ def walk(self, dictobj):
                     self._eventhandler.close_tag(key)
 
             else:
+                nextline = False
                 self._eventhandler.open_tag(key, attrib, hsdattrib)
-                self._eventhandler.add_text(_to_text(value))
+                if isinstance(value, str) and (value.startswith('"<<') or
+                                               value.startswith("'<<") or
+                                               value.startswith("<<")):
```

Comment:
Note, if HSD finds `"<<< hello.hsd"`, it is not supposed to include it. The `<<<` and the `<<+` operators are supposed to be unquoted. (The filename after the operator can be quoted, though.)